actions/workflows/delivery_project_workflow.yaml

```diff
@@ -101,3 +101,15 @@ workflows:
                 delivery_base_api_url: <% $.delivery_service_url %>
                 status_link: <% $.status_link %>
                 skip_mover: <% $.skip_mover %>
+              on-success:
+                - check_ngi_ready_status: <% $.skip_mover = false %>
+
+            check_ngi_ready_status:
```

Comment:
What's the logic behind putting the check of ngi_ready here as a one-time-call? Just a double check that the polling of status in `check_delivery_status` returns the same?